src/IniFileParser/INIFileParser.csproj

```diff
@@ -1,116 +1,34 @@
-    <Compile Include="Model\KeyDataCollection.cs" />
-    <Compile Include="Parser\IniDataParser.cs" />
-    <Compile Include="Model\SectionData.cs" />
-    <Compile Include="Model\SectionDataCollection.cs" />
-    <Compile Include="StreamIniDataParser.cs" />
-    <Compile Include="StringIniParser.cs" />
-    <Compile Include="Exceptions\ParsingException.cs" />
-    <Compile Include="Model\Formatting\DefaultIniDataFormatter.cs" />
-    <Compile Include="Model\Formatting\IIniDataFormatter.cs" />
-    <Compile Include="Model\IniDataCaseInsensitive.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="Parser\ConcatenateDuplicatedKeysIniDataParser.cs" />
-    <Compile Include="Model\Configuration\IniParserConfiguration.cs" />
-    <Compile Include="Model\Configuration\ConcatenateDuplicatedKeysIniParserConfiguration.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="INIFileParser.nuspec" />
-    <None Include="Properties\publickey.snk" />
-  </ItemGroup>
-  <ItemGroup>
-    <BootstrapperPackage Include="Microsoft.Net.Client.3.5">
-      <Visible>False</Visible>
-      <ProductName>.NET Framework 3.5 SP1 Client Profile</ProductName>
-      <Install>false</Install>
-    </BootstrapperPackage>
-    <BootstrapperPackage Include="Microsoft.Net.Framework.3.5.SP1">
-      <Visible>False</Visible>
-      <ProductName>.NET Framework 3.5 SP1</ProductName>
-      <Install>true</Install>
-    </BootstrapperPackage>
-    <BootstrapperPackage Include="Microsoft.Windows.Installer.3.1">
-      <Visible>False</Visible>
-      <ProductName>Windows Installer 3.1</ProductName>
-      <Install>true</Install>
-    </BootstrapperPackage>
-  </ItemGroup>
-  <ItemGroup />
-  <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
-  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
-       Other similar extension points exist, see Microsoft.Common.targets.
-  <Target Name="BeforeBuild">
-  </Target>
-  <Target Name="AfterBuild">
-  </Target>
-  -->
-  <ItemGroup />
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	
+	<PropertyGroup>
+		<TargetFrameworks>net20;netstandard2.0</TargetFrameworks>
+		<RootNamespace>IniParser</RootNamespace>
+		<AssemblyName>INIFileParser</AssemblyName>
+		<PackageReleaseNotes></PackageReleaseNotes>
+		<Description>A .NET, Mono and Unity3d compatible(*) library for reading/writing INI data from IO streams, file streams, and strings written in C#.
+
+Also implements merging operations, both for complete ini files, sections, or even just a subset of the keys contained by the files.
+
+(*) This library is 100% .NET code and does not have any dependencies on Windows API calls in order to be portable.</Description>
+		<Authors>Ricardo Amores Hernández</Authors>
+		<PackageProjectUrl>https://github.com/rickyah/ini-parser</PackageProjectUrl>
+		<PackageLicenseUrl>https://github.com/rickyah/ini-parser/blob/master/LICENSE</PackageLicenseUrl>
```

Comment:
@shuebener To avoid warnings in the latest Visual Studio versions this should be replaced with:
```xml
<PackageLicenseExpression>MIT</PackageLicenseExpression>
```